src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -77,6 +77,11 @@
             <tag name="console.command" command="messenger:consume-messages" />
         </service>
 
+        <service id="console.command.messenger_debug" class="Symfony\Component\Messenger\Command\DebugMessengerCommand">
```

Comment:
All the commands are named `NameDebugCommand`, not `DebugNameCommand`. Could you change it to be something consistent with the rest?

Reply:
All the commands except `DebugAutowiringCommand`.. which i took as a template :sweat_smile:  now updated.